microsoft-edge/devtools-guide-chromium/recorder/index.md

```diff
@@ -0,0 +1,304 @@
+## Open the Recorder tool
+
+1. Open the [TODO app](https://microsoftedge.github.io/Demos/demo-to-do/) in a new tab or window.
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
+
+1. In DevTools, on the **Activity Bar**, select the **Recorder** tab.  If that tab isn't visible, click the **More tools** (![More tools icon](./index-images/more-tools-icon.png)) button, and then select **Recorder**.  The **Recorder** tool opens:
+
+   ![The demo TODO app and the Recorder tool in DevTools, next to it](./index-images/recorder-tool.png)
+
+
+<!-- ====================================================================== -->
+## Record a user flow
+
+To start recording a new user flow:
+
+1. Click **Create a new recording**.
+
+1. Under **Recording name**, enter a name for the recording, such as "Adding TODO tasks".
+
+1. Click **Start recording** at the bottom of the **Recorder** tool:
+
+   ![The recording name and start recording button](./index-images/start-recording.png)
+
+1. In the rendered webpage, interact with the page. For example, enter "task 1" in the **Add a task** input box, and then press **Enter**. Repeat these steps multiple times.
+
+   A red circle appears next to the recording name, indicating that the recording is in progress. The **Recorder** tool records your interactions and displays them as a list of steps:
+
+   ![The recorded user interactions displayed as a list in the Recorder tool](./index-images/list-of-interactions.png)
+
+1. To stop the recording, at the bottom of the list of recorded interactions, click **End recording**.
+
+   The recording stops, and the circle next to the recording name turns blue, indicating that the recording is complete:
+
+   ![The finalized recording and the Replay button](./index-images/final-recording.png)
+
+
+<!-- ====================================================================== -->
+## Record other user flows
+
+To record a new user flow, once you've stopped the recording of the previous user flow, click the **Create a new recording** (![Create a new recording button](./index-images/new-recording-icon.png)) button in the top-left corner of the **Recorder** tool, and then repeat the steps from [Record a user flow](#record-a-user-flow):
+
+![The Create a new recording button](./index-images/create-new-recording.png)
+
+
+<!-- ====================================================================== -->
+## Replay a user flow
+
+You can replay user flows either from the list of recorded user flows, or when viewing the details of a given user flow.
+
+* To view the list of recorded user flows, click the dropdown menu that's in the **Recorder** tool toolbar, at the top, and select **2 recording(s)**. The label might show a different number of recordings, depending on how many recordings you've made.
```

Comment:
can we use a construction like:
Click the dropdown arrow next to **Replay** in the top-right corner of the **Recorder** tool
instead of:
, at the top"